File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/PixelGrabberTest.java
package org.firstinspires.ftc.teamcode.opmodes;

import org.firstinspires.ftc.teamcode.features.PixelGrabber;
import org.firstinspires.ftc.teamcode.internals.registration.OperationMode;
import org.firstinspires.ftc.teamcode.internals.registration.TeleOperation;

/**
 * This tests the manual pixel grabber.
 * <p>
 * Features: {@link org.firstinspires.ftc.teamcode.features.PixelGrabber}
 */
public class PixelGrabberTest extends OperationMode implements TeleOperation {
    @Override
    public void construct() {
        registerFeature(new PixelGrabber());
    }

    @Override
    public void run() {

    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/PixelGrabberTest.kt
package org.firstinspires.ftc.teamcode.opmodes

import org.firstinspires.ftc.teamcode.features.PixelGrabber
import org.firstinspires.ftc.teamcode.internals.registration.OperationMode
import org.firstinspires.ftc.teamcode.internals.registration.TeleOperation

/**
 * This tests the manual pixel grabber.
 *
 *
 * Features: [org.firstinspires.ftc.teamcode.features.PixelGrabber]
 */
class PixelGrabberTest : OperationMode(), TeleOperation {
    override fun construct() {
        registerFeature(PixelGrabber())
    }

    override fun run() {
    }
}
